ui/app/AppLayouts/Profile/views/MessagingView.qml

```diff
@@ -216,7 +216,7 @@ SettingsContentBase {
             Layout.fillWidth: true
             title: qsTr("Contacts, Requests, and Blocked Users")
             requestsCount: root.contactsStore.receivedContactRequestsModel.count
-            sensor.onClicked: Global.changeAppSectionBySectionType(Constants.appSection.profile,
+            onClicked: Global.changeAppSectionBySectionType(Constants.appSection.profile,
```

Comment:
little thing: formatting of the line below could be fixed :)

Reply:
It some kind of `hot fix` I don't want to fix something else in this PR :)